usr.sbin/sysinst/msg.mi.pt

```diff
@@ -0,0 +1,1528 @@
+
+message netdevs
+{Interfaces dispon�veis}
+
+message netinfo
+{To be able to use the network, we need answers to the following:
+
+}
+
+message net_domain
+{Seu dom�nio DNS}
+
+message net_host
+{Seu nome de host}
+
+message net_ip
+{Seu endere�o IPv4}
+
+message net_srv_ip
+{Endere�o IPv4 do servidor}
+
+message net_mask
+{IPv4 Netmask}
+
+message net_namesrv
+{Seu servidor de nomes}
+
+message net_defroute
+{IPv4 gateway}
+
+message net_media
+{Tipo de m�dia de rede (vazio para autoconfigura�ao)}
+
+message net_ssid
+{Wi-Fi SSID?}
+
+message net_passphrase
+{Senha de Wi-Fi?}
+
+message netok
+{A seguir estao os valores que voc� inseriu.
+
+Dom�nio DNS: 		%s
+Nome do host: 		%s
+Servidor de nomes: 	%s
+Interface prim�ria: %s
+Tipo de m�dia:		%s
+IP do host: 		%s
+M�scara de rede: 	%s
+Gateway IPv4: 		%s
+}
+
+message netok_slip
+{Os seguintes sao os valores que voc� inseriu. Eles estao OK?
+
+Dom�nio DNS: 		%s
+Nome do host: 		%s
+Servidor de nomes: 	%s
+Interface prim�ria:	%s
+Tipo de m�dia:		%s
+IP do host:			%s
+IP do servidor:		%s
+M�scara de rede:	%s
+Gateway IPv4:		%s
+}
+
+message netokv6
+{Configura�ao autom�tica IPv6:	%s
+}
+
+message netok_ok
+{Eles estao bem?}
+
+message wait_network
+{
+Aguardando enquanto a interface de rede � ativada.
+}
+
+message resolv
+{Nao foi poss�vel criar /etc/resolv.conf. Instala�ao abortada.
+}
+
+/* Called with: 			Example
+ *  $0 = target prefix			/target
+ *  $1 = error message			No such file or directory
+ */
+message realdir
+{Nao foi poss�vel alterar para o diret�rio $0: $1.
+Instala�ao abortada.}
+
+message delete_xfer_file
+{Excluir ap�s a instala�ao}
+
+/* Called with: 			Example
+ *  $0 = set name			base
+ */
+message notarfile
+{O conjunto de lan�amento $0 nao existe.}
+
+message endtarok
+{Todos os conjuntos de distribui�ao selecionados foram descompactados com sucesso.}
+
+message endtar
+{Houve problemas ao descompactar conjuntos de distribui�ao.
+Sua instala�ao est� incompleta.
+
+Voc� selecionou %d conjuntos de distribui�ao. %d conjuntos nao puderam ser encontrados
+e %d foram ignorados ap�s a ocorr�ncia de um erro. Dos %d
+que foram tentados, %d descompactaram sem erros e %d com erros.
+
+Abortando a instala�ao. Verifique novamente sua fonte de distribui�ao
+e considere reinstalar os conjuntos do menu principal.}
+
+message abort_inst {Instala�ao abortada.}
+message abort_part {Particionamento abortado.}
+
+message abortinst
+{A distribui�ao nao foi carregada com sucesso. Voc� precisar� prosseguir
+manualmente. Instala�ao abortada.
+}
+
+message abortupgr
+{A distribui�ao nao foi carregada com sucesso. Voc� precisar� prosseguir
+manualmente. Atualiza�ao abortada.
+}
+
+message abortunpack
+{Descompactar conjuntos adicionais nao foi bem-sucedido. Voc� precisar�
+continuar manualmente ou escolher uma fonte diferente para conjuntos de lan�amento e tentar
+novamente.
+}
+
+message createfstab
+{TH� um grande problema! Nao � poss�vel criar /mnt/etc/fstab. Saindo!
```

Comment:
I think your finger slipped here and you typed an extra T? Or maybe you forgot to delete a previously existing one?